src/amo/components/AddonReview.js

```diff
@@ -83,25 +83,38 @@ export class AddonReviewBase extends React.Component {
       );
     }
 
+    let placeholder;
+    let prompt;
+    if (review.rating && (review.rating > 3)) {
```

Comment:
Alright. I was nervous about the `&&` being applied first but `5 && 5 = 5` so I guess that's ok!